From cd8d629b66c3bfec4252c5bf2c127b2dcc6c32ae Mon Sep 17 00:00:00 2001
From: Jasper Mattsson <jasu@njomotys.info>
Date: Mon, 10 Sep 2018 21:58:20 +0300
Subject: [PATCH 1/3] clk: mediatek: Register infracfg as a reset controller on
 MT6797

This adds mtk_register_reset_controller to infracfg on the MT6797, to
allow using INFRACFG resets, e.g. for the PMIC wrapper.

Signed-off-by: Jasper Mattsson <jasu@njomotys.info>
---
 drivers/clk/mediatek/clk-mt6797.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/drivers/clk/mediatek/clk-mt6797.c b/drivers/clk/mediatek/clk-mt6797.c
index 20b106764423..e5500de537d9 100644
--- a/drivers/clk/mediatek/clk-mt6797.c
+++ b/drivers/clk/mediatek/clk-mt6797.c
@@ -604,6 +604,8 @@ static int mtk_infrasys_init(struct platform_device *pdev)
 	if (r)
 		return r;
 
+	mtk_register_reset_controller(node, 3, 0x120);
+
 	return 0;
 }
 
-- 
2.18.0

